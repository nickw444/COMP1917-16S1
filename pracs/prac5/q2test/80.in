80
1 8 6 1 6 -10 4 9 10 6 1 9 6 7 3 7 4 4 -5 5 -9 1 8 -7 10 6 10 2 7 10 -4 10 -6 -8 -8 -8 10 2 6 -7 8 -10 2 -7 -6 9 6 7 7 -7 -7 -6 -6 9 -6 -10 6 -3 4 5 -2 -9 1 9 -5 -5 -3 5 3 -2 3 -2 -9 -4 9 -5 1 -6 -5 -8
