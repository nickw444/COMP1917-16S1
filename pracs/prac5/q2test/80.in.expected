average of array: 0.850000
